crml/nft/src/lib.rs

```diff
@@ -801,6 +1040,46 @@ impl<T: Config> Module<T> {
 			<TokenOwner<T>>::insert((token_id.0, token_id.1), token_id.2, new_owner);
 		}
 	}
+	/// Mint additional tokens in a series
+	fn do_mint_additional(
+		owner: &T::AccountId,
+		collection_id: CollectionId,
+		series_id: SeriesId,
+		serial_number: SerialNumber,
+		quantity: TokenCount,
+	) -> DispatchResult {
+		ensure!(quantity > Zero::zero(), Error::<T>::NoToken);
+		ensure!(
+			!Self::is_single_issue(collection_id, series_id),
+			Error::<T>::AddToUniqueIssue
+		);
+
+		// Mint the set tokens
+		for serial_number in serial_number..serial_number + quantity {
+			<TokenOwner<T>>::insert((collection_id, series_id), serial_number as SerialNumber, &owner);
+		}
+
+		SeriesIssuance::mutate(collection_id, series_id, |q| *q = q.saturating_add(quantity));
```

Comment:
this is a good example of where mutate is used well

Reply:
Oooooh I get it, so just replacing the value entirely we use insert, but if we need to read and write mutate. Like you've been saying this whole time 😅